src/components/CompleteProfile/CompleteProfile.tsx

```diff
@@ -0,0 +1,399 @@
+	};
+
+	const handleAddSocialAccount = (socialAccountToAdd: string) => {
+		setSocialAccountInputs((previousAccounts) => {
+			if (!previousAccounts?.includes(socialAccountToAdd)) {
+				return [...previousAccounts, socialAccountToAdd];
+			}
+			return previousAccounts;
+		});
+		toggleIconVisibility(socialAccountToAdd);
+	};
+
+	const handleRemoveSocialAccount = (socialAccountToRemove: string) => {
+		setSocialAccountInputs((previousAccounts) => previousAccounts.filter((socialAccount: string) => socialAccount !== socialAccountToRemove));
+		toggleIconVisibility(socialAccountToRemove);
+	};
+
+	return (
+		<>
+			<StepBar
+				currentStep={3}
+				steps={[
+					{ label: 'Account confirmed' },
+					{ label: 'Check the conduct code' },
+					{ label: 'Complete your profile' }
+				]}
+			/>
+			<form action='' encType='multipart/form-data' id='complete-profile-form'>
+				<h2>Complete your profile</h2>
+
+				<div id='fields-wrapper'>
+					<details open>
+						<summary className='text-h6'>
+							<span>In a nutshell:</span>
+						</summary>
+
+						<div className='details-content-wrapper'>
+							<div className='details-nutshell-grid'>
+								{/* TODO: Create resuable input components later */}
+								<div>
+									<label htmlFor='name'>Name</label>
+									<input
+										className='text-input'
+										id='name'
+										name='name'
+										type='text'
+									/>
+								</div>
+								<div>
+									<label htmlFor='email' className='input-required'>
+										E-mail
+										<span>REQUIRED</span>
+									</label>
+									<input
+										className='text-input'
+										id='email'
+										name='email'
+										type='email'
+										required
+									/>
+								</div>
+								<div>
+									<label htmlFor='slack-handle' className='input-required'>
+										Slack handle
+										<span>REQUIRED</span>
+									</label>
+									<input
+										className='text-input'
+										id='slack-handle'
+										name='slack-handle'
+										type='text'
+										aria-description='Your slack handle to TorontoJS'
+										placeholder='Your slack handle to TorontoJS'
+										required
+									/>
+								</div>
+								<div>
+									<label htmlFor='pronouns'>Pronouns</label>
+									<input
+										className='text-input'
+										id='pronouns'
+										name='pronouns'
+										type='text'
+										list='pronouns-options'
+										placeholder='Your pronouns (optional)'
+									/>
+									<datalist id='pronouns-options'>
+										<option>He/him</option>
+										<option>She/her</option>
+										<option>They/them</option>
+									</datalist>
+								</div>
+								<div>
+									<span>Date of birth</span>
+									<br />
+									<div className='dob-wrapper'>
+										<label htmlFor='month'>Month</label>
+										<select id='month' name='month'>
+											<option value='01'>January</option>
```

Comment:
```suggestion
											<option selected disabled hidden value=''>Select a month</option>
											<option value='01'>January</option>
```

This is a neat trick to add an option that doesn't show up but still adds a "placeholder" for the user.